zmdocument.cls

```diff
@@ -220,23 +230,39 @@
 \newcommand{\logo}[1]{\def\@logo{#1}}
 \newcommand{\website}[1]{\def\@website{#1}}
 \newcommand{\editor}[1]{\def\@editor{#1}}
+\newcommand{\editorlogo}[1]{\def\@editorlogo{#1}}
 \newcommand{\authorlink}[1]{\def\@authorlink{#1}}
 
 \renewcommand{\maketitle}{%
+    \clearpage
+    \thispagestyle{empty}
+    \pagecolor{bgtitle}
+    \color{texttitle}
     \ifcsdef{@authorlink}{}{\authorlink{https://zestedesavoir.com/membres/voir}}
     \ifcsdef{@website}{}{\website{https://zestedesavoir.com}}
-    \ifcsdef{@editor}{}{\editor{Zeste de Savoir}}
+    \ifcsdef{@editor}{}{\editor{https://zestedesavoir.com}}
+    \ifcsdef{@editorlogo}{}{\editorlogo{zestedesavoir.png}}
     \ifcsdef{@logo}{}{\logo{default_logo.png}}
-    {\centering
-    \vspace*{0.5 cm}
-    \includegraphics[width=\linewidth]{\@logo}\\[1.0 cm]
-    \LARGE \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\bsc{##1}} \qquad}
-           \expandafter\docsvlist\expandafter{\@author}\\[2.0 cm]
-    \Large \href{\@website}{\@editor}\\[0.5 cm]
+    {
+    \raggedleft
+    \Large\bf \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\bsc{##1}}\\}
+           \expandafter\docsvlist\expandafter{\@author}\\[1.0 cm]
+    \centering
+    \vspace*{2.5 cm}
+    \includegraphics[height=4cm]{\@logo}\\
```

Comment:
To avoid any issue, I would use a `max size` here (see [this example](https://github.com/zestedesavoir/latex-template/pull/34/files#diff-b4771c7685c0b1af5bc9082b5dd4e317R250)). In the (possible) option when someone would like to use our template, this 4cm default could be an issue (if, for example, their logo is longer than larger).